Makefile

```diff
@@ -122,6 +122,10 @@ help:
 	@echo '    Documentation:'
 	@echo '        docs              - preview documentation website'
 	@echo '        metrics-docs      - generate metrics reference documentation page'
+	@echo 'End-to-end tests: '
+	@echo '    e2e-test                                        - run e2e tests'
+	@echo '    e2e-test E2E_BUILD_IMAGES=0                     - run e2e tests without (re-)building images'
+	@echo '    e2e-test E2E_TESTS=./tests/e2e/tests/skeleton   - run a specific e2e test'
```

Comment:
I was just thinking we should do something like https://www.thapaliya.com/en/writings/well-documented-makefiles/, because many targets are missing from this help. But I'm not sure how to best document multiple options for one target then. (I mean, it's very helpful to have options documented, I'm just thinking out loud here)

Reply:
I guess we could something like
```
e2e-tests: ## run e2e tests, use E2E_BUILD_IMAGES=0 to avoid running images and E2E_TESTS=./tests/e2e/tests/skeleton ro run only the skeleton test
```